src/components/Match/matchColumns.jsx

```diff
@@ -50,8 +50,14 @@ export const heroTdColumn = {
 };
 
 const partyStyles = (row, match) => {
-  if (match.players && match.players.filter(x => x.party_size > 1).length) {
-    return <div className={`group${row.party_id}`} />;
+  if (match.players && row.party_size > 1) {
+    const partyIndex = lodash.compose(
```

Comment:
avoid using lodash functions if possible to reduce bundle size.

Reply:
It was already being imported in the file. I can try importing the functions individually.

Reply:
I think because of tree shaking the bundle size is smaller if the functions aren't used.  I'd like to move the entire codebase away from lodash eventually, but haven't had the time to do so.